src/factories/schema-registry-factory.js

```diff
@@ -618,12 +631,25 @@ var SchemaRegistryFactory = function ($rootScope, $http, $location, $q, $log, Ut
       var sortedHistory = UtilsFactory.sortByVersion(subjectHistory);
       for (var i = 0; i < sortedHistory.length; i++) {
         var previous = '';
+
         if (i > 0)
```

Comment:
You changed the body of if and made it multiline you need to add { and } around it.